src/test/java/seedu/address/logic/commands/RemoveCommandTest.java

```diff
@@ -0,0 +1,80 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.showPersonAtIndex;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.RemoveCommand.RemovePersonDescriptor;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.testutil.RemovePersonDescriptorBuilder;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for Remove Command.
+ */
+public class RemoveCommandTest {
+
+    private static final Index INDEX_FIRST_ITEM = Index.fromOneBased(1);
+
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    public void setFullView() {
+        model.setFullView();
+    }
+
+    public void setDayView() {
+        model.setDayView();
+    }
+
+    public void setListView() {
+        model.setListView();
+    }
```

Comment:
One other test that you could include could be execute_noFieldSpecifiedUnfilteredList_success, which is to check whether removeCommand will return error message when there is no field specified.

Reply:
yessir i will try my best